src/components/table/table.tsx

```diff
@@ -53,6 +53,13 @@ export class Table {
     @Prop()
     public totalRows: number;
 
+    /**
+     * The initial sorted columns
+     */
+
+    @Prop()
+    public defaultSortedColumns: Tabulator.Sorter[];
```

Comment:
I don't think we should name it like this, something like `sorting` would be better. We might want to add support for changing the sorting by changing this property in the future and then the name `sorting` makes more sense.

Also, we cannot expose `Tabulator.Sorter` to consumers since that is internal to the implementation of the `limel-table` component. There is an interface named `ColumnSorter` in `src/components/table/table.types.ts`, use that one instead :slightly_smiling_face: